src/test/java/com/google/sps/servlets/IntervieweeFeedbackServletTest.java

```diff
@@ -64,6 +67,11 @@
   @Before
   public void setUp() {
     helper.setUp();
+    try {
+      emailSender = new FakeEmailSender(new Email("interviewme.business@gmail.com"));
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
```

Comment:
Why do we need a try catch here?

Reply:
The constructor for the fakeEmailSender and SendgridEmailSender throws an IOException